models/points.go

```diff
@@ -25,8 +25,20 @@ var (
 		' ': []byte(`\ `),
 		'=': []byte(`\=`),
 	}
+
+	// used to track length of binary-encoded time.Time
+	timeLen int
 )
 
+func init() {
+	binTime, err := time.Now().MarshalBinary()
+	if err != nil {
+		// we'll be marshaling time objects a lot
+		panic(err)
+	}
+	timeLen = len(binTime)
```

Comment:
I could marshal the time first and use the `len()` of the resulting data to allocate the final slice, though.